ivadomed/main.py

```diff
@@ -852,8 +858,13 @@ def cmd_test(context):
 
 
 def cmd_eval(context):
+    path_pred = os.path.join(context['log_directory'], 'pred_masks')
+    if not os.path.isdir(path_pred):
+        print('\nRun Inference\n')
+        cmd_test(context)
+    print('\nRun Evaluation on {}\n'.format(path_pred))
```

Comment:
If the inference has not been run before running the evaluation --> then we call `cmd_test`.

Which means that we only have to run `cmd_eval` if `pred_masks` folder does not exist: "kill two birds with one stone" JCA.